include/wx/msw/wx.rc

```diff
@@ -135,14 +115,9 @@ wxBITMAP_STD_COLOURS    BITMAP "wx/msw/colours.bmp"
 
 #define wxRC_STR(text) wxRC_STR2(text)
 #define wxRC_STR2(text) #text
-#define wxRC_CONCAT(a, b, c) wxRC_CONCAT2(a, b, c)
-#define wxRC_CONCAT2(a, b, c) a ## b ## c
 
-#ifdef __GNUC__
-    #define wxMANIFEST_FILE "wx/msw/" wxRC_STR(wxMANIFEST_CPU) wxRC_STR(wxMANIFEST_DPI)
-#else
-    #define wxMANIFEST_FILE wxRC_CONCAT(wx/msw/, wxMANIFEST_CPU, wxMANIFEST_DPI)
-#endif
+#define wxMANIFEST_FILE "wx/msw/wx" wxRC_STR(wxMANIFEST_DPI)
+
```

Comment:
I am not sure why there is a difference between GCC and MSVC: `wxRC_STR `vs w`xRC_CONCAT`?

Reply:
It looks like windres wants to have a string while rc wants to have a path, but it's indeed a bit weird and it could be worth testing if both of them might accept the same form.

OTOH it's a pretty recent change done in cd1d15351c (Fix concatenating manifest name in resource file, 2019-12-13) so it probably not only was, but still is, necessary as otherwise @MaartenBent wouldn't have done it.

Reply:
Sorry, I didn't actually see the diff, so I'd just like to make it clear that I think this change should be reverted, i.e. `#ifdef __GNUC__` restored, unless we are _really_ certain that strings work with all MSVC versions.

Reply:
[PR1678 ](https://github.com/wxWidgets/wxWidgets/pull/1678) contains some more details for this change and a link to the report of strings not working for MSVC.